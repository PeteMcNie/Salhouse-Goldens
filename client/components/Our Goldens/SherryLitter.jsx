import React from 'react'

// import Image from 'react-bootstrap/Image'

function SherryLitter () {
  return (
    <>
      <div>
        <h2>Sherry&apos;s Litter 2006</h2>
        <h1>Copy Template from Kula&apos;sLitter</h1>
      </div>
    </>

  )
}

export default SherryLitter
