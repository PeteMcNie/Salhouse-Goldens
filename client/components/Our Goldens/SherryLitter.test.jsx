import React from 'react'
import { render } from '@testing-library/react'

import SherryLitter from './SherryLitter'

test('Sherry component matches snapshot', () => {
  const { container } = render(<SherryLitter />)
  expect(container).toMatchSnapshot()
})
