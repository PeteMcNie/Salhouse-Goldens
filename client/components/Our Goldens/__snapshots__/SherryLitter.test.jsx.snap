// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sherry component matches snapshot 1`] = `
<div>
  <div>
    <h2>
      Sherry's Litter 2006
    </h2>
    <h1>
      Copy Template from Kula'sLitter
    </h1>
  </div>
</div>
`;
